tests/test_conversion/test_expressions.py

```diff
@@ -686,6 +686,12 @@ def test_sub_s(input, expected_result, expectation, block):
         no_exception(),
         hcl2.Variable("foo", {"value": "bar"}),
     ),
+    (
+        ["bash $foo and Cloudformation ${foo}", {"foo": "bar"}],
+        "bash $foo and Cloudformation bar",
```

Comment:
```suggestion
        ["bash $foo and Cloudformation ${foo}", {"foo": "var.bar"}],
        "bash $foo and Cloudformation ${var.bar}",
```
Forgot how this function even works 🙄 , fixed now 🤞